src/deployment/registration.py

```diff
@@ -232,7 +232,9 @@ def add_application_password(app_object_id: UUID) -> Tuple[str, str]:
     while count < tries:
         count += 1
```

Comment:
Could we add a log message to this loop along the lines of "retrying app password creation", so if you get the (now-nicer) warning message, it also tells you that it is retrying before you get the success messsage?

Reply:
added.